integration_tests/steps/integration_tests.py

```diff
@@ -693,40 +693,39 @@ def test_output(context, scenario, sim_duration, slot_length, tick_length):
 @then('the energy bills report the correct accumulated traded energy price')
 def test_accumulated_energy_price(context):
     bills = context.simulation.endpoint_buffer.market_bills.bills_results
-    cell_tower_bill = bills["Cell Tower"]["earned"] - bills["Cell Tower"]["spent"]
-    net_traded_energy_price = cell_tower_bill
+    # cell_tower_bill = bills["Cell Tower"]["earned"] - bills["Cell Tower"]["spent"]
+    # net_traded_energy_price = cell_tower_bill
     for house_key in ["House 1", "House 2"]:
         extern_trades = bills[house_key]["External Trades"]
         assert extern_trades["total_energy"] == extern_trades["bought"] - extern_trades["sold"]
         assert extern_trades["total_cost"] == extern_trades["spent"] - extern_trades["earned"]
-        house_bill = \
-            bills[house_key]["Totals"]["spent"] - \
-            bills[house_key]["Totals"]["earned"] + \
-            bills[house_key]["Accumulated Trades"]["earned"] - \
-            bills[house_key]["Accumulated Trades"]["spent"]
-
-        area_net_traded_energy_price = \
-            sum([v["earned"] - v["spent"] for k, v in bills[house_key].items()
-                if k not in ACCUMULATED_KEYS_LIST])
-        assert isclose(area_net_traded_energy_price, house_bill, rel_tol=1e-02), \
-            f"area: {area_net_traded_energy_price} house {house_bill}"
-        net_traded_energy_price += area_net_traded_energy_price
-
-        for accumulated_section in ["Accumulated Trades", "External Trades",
-                                    "Market Fees", "Totals"]:
+        # house_bill = \
```

Comment:
Can you please add a comment for future to remember the context of downgrading this integration_test step